app/views/dashboard/index.html.erb

```diff
@@ -1,5 +1,36 @@
-<% if current_user %>
- <h4> <%= current_user.name %>'s Dashboard</h4>
- <% else %>
- <h4> Guest Dashboard </h4>
- <% end %>
\ No newline at end of file
+
+<div class="row">
+</div>
+ <div class="row">
+   <div class ="col s4">
+     <%= image_tag(@user.avatar_url) %>
+   </div>
+   <div class="col s2">
+      Repositories: <%= @user.repos(@user.token).count %>
```

Comment:
@caward12 Since `@user` gets initialized with the token, we should be able to call `@user.repos` and avoid passing in the token again as a parameter. This applies to all of the method calls on `@user` in here that take the token as a parameter.